mrpt_graphslam_2d/remoteApi/Makefile

```diff
@@ -0,0 +1,237 @@
+#############################################################################
```

Comment:
Why is the Makefile added to git? Shouldn't this be covered by cmake?

Reply:
I forgot to add Makefile to the gitignore. I'll add all the unnecessary files to the gitignore and send a commit